packages/shorebird_code_push_client/test/src/code_push_client_test.dart

```diff
@@ -185,15 +185,64 @@ void main() {
 
         final request = verify(() => httpClient.send(captureAny()))
             .captured
-            .single as http.BaseRequest;
+            .single as http.MultipartRequest;
+
         expect(request.method, equals('POST'));
+        expect(request.fields['arch'], equals(arch));
+        expect(request.fields['platform'], equals(platform.name));
+        expect(request.fields['hash'], equals(hash));
+        expect(request.fields['size'], equals(fixture.lengthSync().toString()));
+        expect(request.fields.containsKey('hash_signature'), isFalse);
+
         expect(
           request.url,
           equals(v1('apps/$appId/patches/$patchId/artifacts')),
         );
         expect(request.hasHeaders(expectedHeaders), isTrue);
       });
 
+      group('when a hash signature is informed', () {
```

Comment:
"when a hash signature is provided" maybe? I'm not sure what "informed" means here

Reply:
Yeah, "provided" seems better! Maybe I have been using "informed" wrong in english. In portuguese, the literal translation of informed, which is "informado", is a direct alternative for provided when "the thing that were provided" is an information/data.